.github/workflows/CI.yml

```diff
@@ -4,9 +4,17 @@ on:
   pull_request:
 name: CI
 jobs:
-  testing:
-    runs-on: ubuntu-latest
-    name: testing
+  flatpak:
+    name: "Flatpak"
+    runs-on: ubuntu-22.04
+    container:
+      image: bilelmoussaoui/flatpak-github-actions:gnome-44
+      options: --privileged
     steps:
-    - uses: actions/checkout@master
-    - uses: tchx84/gtk-apps-testing-docker-action@gjs
+    - uses: actions/checkout@v3
+    - uses: flatpak/flatpak-github-actions/flatpak-builder@v5
+      with:
+        bundle: "flatseal.flatpak"
+        manifest-path: "com.github.tchx84.Flatseal.json"
+        run-tests: "true"
+        cache-key: flatpak-builder-${{ github.sha }}
```

Comment:
The tests are "passing"  because jasmine-gjs is not included here, and therefore the actual tests are not running. 

I am working on this here https://github.com/tchx84/Flatseal/pull/514 it has it owns issues. For this PR we should aim to have the tests pass with the current CI setup.

You should be able to run this with a fedora toolbox, installing these dependencies https://github.com/tchx84/gtk-apps-testing-docker-action/blob/gjs/Dockerfile and doing this https://github.com/tchx84/gtk-apps-testing-docker-action/blob/gjs/entrypoint.sh

Reply:
I am trying but no idea how it works. It might be easier to just update the container in use for now

Reply:
Just to make sure there's no misunderstanding here: 

All this change is doing is simply disabling the tests. That's not a solution.

That GH action is not working properly either, reason why I am investigating it on https://github.com/tchx84/Flatseal/pull/514 , so let's separate that issue from this PR. This PR is big enough as it is. So,

**Please remove this change**, or we're are simply going to get stuck here. 

I suggest you try:

1. Creating a Fedora tool
2. Installing the dependencies as per https://github.com/tchx84/gtk-apps-testing-docker-action/blob/gjs/Dockerfile (copy paste)
3. Run "ninja test"

If there's something that is not working or clear, just ask specific questions.